dev-libs/sway/sway-1.0_beta2.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == 9999 ]] ; then
+		EGIT_REPO_URI="https://github.com/swaywm/sway.git"
+		inherit git-r3
+else
+		# Version format: major.minor-beta.betanum
+		SWAY_PV="$(ver_cut 1-2)-$(ver_cut 3).$(ver_cut 4)"
+		SRC_URI="https://github.com/swaywm/sway/archive/${SWAY_PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
`${PN}`

Reply:
The name should contain also the package version, not only the package name (`${P}=${PN}-${PV}`). This is the approach used in all ebuilds that need to rename the archive. Try with `grep -r '.tar.gz -> ${P}.tar.gz' /usr/portage/`.